releases/forecastle.yaml

```diff
@@ -39,6 +40,7 @@ releases:
       - {{ env "FORECASTLE_CUSTOM_APPS_YAML" }}
       {{- end }}
       - forecastle:
+          createCustomResource: false
```

Comment:
Is this a cold start flag? If so, it needs to be an env.

Reply:
it not for cold start but to avoid reacting main forecastle on unlimited staging CRDs
any way I've made it env